.github/workflows/wheels.yml

```diff
@@ -1,12 +1,11 @@
 name: Build wheels
 
 on:
-  # TODO" delete pull_request below when this is working
-  pull_request:
```

Comment:
The way we made it for `lru-dict` was on push and pull_request.
https://github.com/amitdev/lru-dict/blob/v1.1.8/.github/workflows/build-and-deploy.yml#L3
Because you still want to see if people didn't break wheel building on pull request.
Then the PyPI upload has a specific check for tagging only (I can understand why you may want an extra regex here too).
https://github.com/amitdev/lru-dict/blob/v1.1.8/.github/workflows/build-and-deploy.yml#L40

Reply:
Thanks for the feedback.  Yeah, that makes sense.  I've gone back and forth on this.  It's about who has the burden of responsibility to make sure wheels pass.  If they fail, I think it's more likely that something changed with cibuildwheel or GA than being caused by the PR.  In this case, I don't want to frustrate contributors via long CI cycles _and_ having them worry about wheels.

The current strategy in this PR is:
- Build wheels for all commits to main branch and for release tags, but not for PRs
  - Hence, we'll know if a PR breaks wheels after it's merged; I'll investigate and ping the author if appropriate
- Upload all wheel and sdist artifacts to github as part of GA and retain for 7 days
- Upload wheel and sdist to PyPI for tags only

I'd like to try this for now, and I'm open to changing strategies if we find that PRs sometimes cause wheels to fail.

Reply:
Oh I didn't think of this reason/approach, that makes sense too indeed.
Thanks for sharing